ush/forecast_postdet.sh

```diff
@@ -0,0 +1,961 @@
+    use_hydro_pressure=".false."   # use hydrostatic pressure for physics
+    if [ $warm_start = ".true." ]; then
+      make_nh=".false."              # restarts contain non-hydrostatic state
+    else
+      make_nh=".true."               # re-initialize non-hydrostatic state
+    fi
+  
+  else # hydrostatic options
+    hydrostatic=".true."
+    phys_hydrostatic=".false."     # ignored when hydrostatic = T
+    use_hydro_pressure=".false."   # ignored when hydrostatic = T
+    make_nh=".false."              # running in hydrostatic mode
+  fi
+  
+  # Conserve total energy as heat globally
+  consv_te=${consv_te:-1.} # range 0.-1., 1. will restore energy to orig. val. before physics
+  
+  # time step parameters in FV3
+  k_split=${k_split:-2}
+  n_split=${n_split:-6}
+  
+  if [ $(echo $MONO | cut -c-4) = "mono" ];  then # monotonic options
+    d_con=${d_con_mono:-"0."}
+    do_vort_damp=".false."
+    if [ ${TYPE} = "nh" ]; then # non-hydrostatic
+      hord_mt=${hord_mt_nh_mono:-"10"}
+      hord_xx=${hord_xx_nh_mono:-"10"}
+    else # hydrostatic
+      hord_mt=${hord_mt_hydro_mono:-"10"}
+      hord_xx=${hord_xx_hydro_mono:-"10"}
+    fi
+  
+  else # non-monotonic options
+    d_con=${d_con_nonmono:-"1."}
+    do_vort_damp=".true."
+    if [ ${TYPE} = "nh" ]; then # non-hydrostatic
+      hord_mt=${hord_mt_nh_nonmono:-"5"}
+      hord_xx=${hord_xx_nh_nonmono:-"5"}
+    else # hydrostatic
+      hord_mt=${hord_mt_hydro_nonmono:-"10"}
+      hord_xx=${hord_xx_hydro_nonmono:-"10"}
+    fi
+  fi
+  
+  if [ $(echo $MONO | cut -c-4) != "mono" -a $TYPE = "nh" ]; then
+    vtdm4=${vtdm4_nh_nonmono:-"0.06"}
+  else
+    vtdm4=${vtdm4:-"0.05"}
+  fi
+  
+  if [ $warm_start = ".true." ]; then # warm start from restart file
+    nggps_ic=".false."
+    ncep_ic=".false."
+    external_ic=".false."
+    mountain=".true."
+    if [ $read_increment = ".true." ]; then # add increment on the fly to the restarts
+      res_latlon_dynamics="fv3_increment.nc"
+    else
+      res_latlon_dynamics='""'
+    fi
+  
+  else # CHGRES'd GFS analyses
+    nggps_ic=${nggps_ic:-".true."}
+    ncep_ic=${ncep_ic:-".false."}
+    external_ic=".true."
+    mountain=".false."
+    read_increment=".false."
+    res_latlon_dynamics='""' 
+  fi
+  
+  # Stochastic Physics Options
+  if [ ${SET_STP_SEED:-"YES"} = "YES" ]; then
+    ISEED_SKEB=$((CDATE*1000 + MEMBER*10 + 1))
+    ISEED_SHUM=$((CDATE*1000 + MEMBER*10 + 2))
+    ISEED_SPPT=$((CDATE*1000 + MEMBER*10 + 3))
+  else
+    ISEED=${ISEED:-0}
+  fi
+  DO_SKEB=${DO_SKEB:-".false."}
+  DO_SPPT=${DO_SPPT:-".false."}
+  DO_SHUM=${DO_SHUM:-".false."}
+  JCAP_STP=${JCAP_STP:-$JCAP_CASE}
+  LONB_STP=${LONB_STP:-$LONB_CASE}
+  LATB_STP=${LATB_STP:-$LATB_CASE}
+  
+  #------------------------------------------------------------------
+  # make symbolic links to write forecast files directly in memdir
+  cd $DATA
+  if [ $RUN_CCPP = "YES" ]; then
```

Comment:
This seems out of date with f/c-c 
It's my understanding that RUN_CCPP is now always YES so we shouldn't need this If statement here.

Reply:
I know that is true for the coupled model, but what about the develop side? @KateFriedman-NOAA @WalterKolczynski-NOAA May I ask you this question: Is RUN_CCPP always YES? on the develop side?

Reply:
@JianKuang-UMD The develop branch sets RUN_CCPP=YES by default but I removed that variable in my commit to feature/coupled_sprint. It's my understanding that we'll be running CCPP always...can anyone confirm otherwise? We may not need RUN_CCPP or CCPP toggles anymore.

Reply:
Yes, CCPP will always be on now. The build script automatically uses it now.

Reply:
Some of the changes I made to the forecast script also look at the CCPP suite definition file to determine settings.

Reply:
Thanks for your comments, I removed RUN_CCPP from the scripts